.github/workflows/main.yml

```diff
@@ -216,14 +216,15 @@ jobs:
             --release --processes $(nproc) --timeout-multiplier 2 \
             --total-chunks ${{ env.max_chunk_id }} --this-chunk ${{ matrix.chunk_id }} \
             --log-raw test-wpt.${{ matrix.chunk_id }}.log \
-            --filter-intermittents=filtered-wpt-results.${{ matrix.chunk_id }}.json
+            --filter-intermittents=filtered-test-wpt.${{ matrix.chunk_id }}.json
       - name: Archive filtered results
         uses: actions/upload-artifact@v3
         if: ${{ always() }}
         with:
           name: wpt-filtered-results-linux
           path: |
-            filtered-wpt-results.${{ matrix.chunk_id }}.json
+            filtered-test-wpt.${{ matrix.chunk_id }}.json
+            unexpected-test-wpt.${{ matrix.chunk_id }}.log
```

Comment:
without reading the code, it’s not really clear where this file comes from. wdyt about lifting the `unexpected-{raw_log_file}` decision out to main.yml via a command line option?

Reply:
This is a good point. I've plumbed this out into a command-line argument. It should be a bit clearer now.